fs/metrics/common/metrics.go

```diff
@@ -71,6 +71,21 @@ const (
 	SynchronousReadRegistryFetchCount = "synchronous_read_remote_registry_fetch_count" // TODO revisit (wrong place)
 	SynchronousBytesServed            = "synchronous_bytes_served"
 
+	// fuse operation failure metrics
+	// - node level
+	FuseGetattrFailureCount   = "fuse_getattr_failure_count"
+	FuseGetxattrFailureCount  = "fuse_getxattr_failure_count"
+	FuseListxattrFailureCount = "fuse_listxattr_failure_count"
+	FuseLookupFailureCount    = "fuse_lookup_failure_count"
+	FuseOpenFailureCount      = "fuse_open_failure_count"
+	FuseReadlinkFailureCount  = "fuse_readlink_failure_count"
+	FuseReaddirFailureCount   = "fuse_readdir_failure_count"
+	FuseStatfs                = "fuse_statfs_failure_count"
```

Comment:
Why does ``FuseStatfs`` not follow the name``*FailureCount`` ?

Reply:
I see it's logged for every ``Statfs`` operation. I suggest removing this metric as this PR is about fuse failure metrics.